nullaway/src/test/java/com/uber/nullaway/NullAwayAndroidTest.java

```diff
@@ -11,7 +11,6 @@
 
 /** Unit tests for {@link com.uber.nullaway.NullAway}. */
 @RunWith(JUnit4.class)
-@SuppressWarnings("CheckTestExtendsBaseClass")
```

Comment:
Unrelated to this PR?  In any case I agree with removing

Reply:
Unrelated, but long time unnecessary since outside of Uber's codebase 😅